### DISCUSSION TITLE
How can I get knowledge of code workflow as I am willing to contribute to the project.

### DISCUSSION THREAD

**Discussion by nikhilrajgarhia** [testcontainers/testcontainers-java#7783] [Q&A]
Hi All, I am interested in this project and willing to contribute in future. For this, I am looking on how to understand the code and important areas to solidify my understanding. Do we have any explanatory video or docs which I can go through. I have been through this [doc](https://java.testcontainers.org/)
P.S: This would be my first contribution, any detailed doc/walkthrough would be very helpful. Thanks in advance.

**Comment by kiview** [comment]
Hey @nikhilrajgarhia, there are no specific onboarding guides or docs for new contributors. The best approach to get more familiar with the project as a contributor would be joining [Slack](https://slack.testcontainers.org/) and helping out with answering questions and support requests by users there in the `tc-java` channel. Alternatively, you can also take a look at our open [issues](https://github.com/testcontainers/testcontainers-java/issues) and help with triaging them. 

Either way, before starting work on any substantial contributions, it is advised to get in touch with the maintainers first, either through an GH issue, discussion, or via Slack.

**Reply by nikhilrajgarhia** [reply]
Hi @kiview , thanks for your response. I am able to check the contributors on '<>code' page. Can I reach anyone of them? or if you can help me with few maintainers to whom I can reach in case of any clarifications.
